recipes/radtraq/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "radtraq" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/RadTraQ-{{ version }}.tar.gz
+  sha256: ccf79c40658b0cf3acc790a61d1b825c6411c5f291a107bb93a8512333e5536b
+  patches:
+    - 001_versioneer_configparser.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+  run:
+    - python >=3.8
+    - pandas <2.0
+    - act-atmos
+    - numpy
+    - scipy
+    - xarray
+    - matplotlib-base
+    - netcdf4
+    - pint
```

Comment:
```suggestion
    - pint
    - setuptools
```

Reply:
Erm, I believe the error is from using an old version of versioneer unsupported by python 3.11 ???